azure-pipelines.yml

```diff
@@ -42,9 +42,11 @@ jobs:
           command: 'test'
           arguments: '-v ./... -cover'
           workingDirectory: '$(System.DefaultWorkingDirectory)'
+  - job: 
+    displayName: integration-test-org1-and-org2
+    steps:
+      - script:  ./integration/integration-test-org1-and-org2.sh 
   - job:
-    displayName: integration-test
+    displayName: integration-test-select-endorser
     steps:
-      - script: ./integration-test.sh
-
-
+      - script: ./integration/integration-test-select-endorser.sh
```

Comment:
this could actually take cli args, no? we could *reuse* the same script (possibly take in some more args)

Reply:
for this one, I suppose we should support for org1 & org2 and org1 or org2 as two test cases.
So 1st of all, there should be two cases, as phase of azp.
I will change it to pass parameters.